### PR TITLE
Fonts styling

### LINKED ISSUE #91: Integrate SciFly Font
As a user, when I visit any view, I want to see all text in the SciFly font.

- [ ] Update to README (FE screenshots)

### LINKED ISSUE #95: Give 7 day forcast a more readable background
Give 7 day forcast a more readable background

- [ ] Happy/sad path testing

- [ ] Update to README (FE screenshots, BE API endpoint documented)

### LINKED ISSUE #96: readability of weather show page
Improve readability of weather on show page (weather icon over text is difficult to read)

### PR DESCRIPTION
**Pull request by nmking22**
This PR will:
* Update all text with Gagalin imported font
* Add styling on all views
* Update flights show weather display so that images are viewable

- [x] Repo Representation: Front End

- [x] All Tests are passing

- [ ] Updated README.md with added information - readme gifs not updated

Resolves #95 
Resolves #96 
Resolves #91 

Screen Shot 2021-01-22 at 1 30 15 PM
Screen Shot 2021-01-22 at 1 30 34 PM
Screen Shot 2021-01-22 at 1 30 56 PM
Screen Shot 2021-01-22 at 1 31 15 PM
Screen Shot 2021-01-22 at 1 31 31 PM
Screen Shot 2021-01-22 at 1 32 01 PM
Screen Shot 2021-01-22 at 1 32 24 PM
Screen Shot 2021-01-22 at 1 32 39 PM

### CODE REVIEW

**Review by SageOfCode — APPROVED** [review]
This looks great, good work! I can't wait to see it in production.

**Review by stillsheryl — APPROVED** [review]
Great additions! The weather on the show page is SO much more readable!

File: app/views/flights/show.html.erb [hunk 1]
```diff
@@ -6,54 +6,72 @@
...
       <p class="TripBooking">
         <%= link_to("Book Your Flight", @trip.booking_link, target: '_blank', class: 'myButton') %></p>
     <% else %>
       <p>Please <%= link_to "login with Google", google_login_path %> to book this flight!</p>
     <% end %>
+    <br>
+  </div><br>
+
+  <div class ='weather-header'>
+    <h3>This Week's Weather</h3>
```
Comment (JesseMellinger) [inline app/views/flights/show.html.erb:50]: The addition of this header looks great!

File: app/views/flights/show.html.erb [hunk 2]
```diff
@@ -6,54 +6,72 @@
...
           <p>Feels like: <%= day.day_feels_like_f.round(0)%>°</p>
           <p>Hi: <%= day.max_f.round(0)%>° Lo: <%=day.min_f.round(0)%>°</p>
         </div>
       </div>
     </div>
     <% end %>
   </div>
+
+  <div class ='weather-header'>
+    <h3>Book Accomodations</h3>
```
Comment (JesseMellinger) [inline app/views/flights/show.html.erb:72]: I think we're missing an 'm' here 👍🏻

**Review by JesseMellinger — APPROVED** [review]
Styling looks great and I think the new font is definitely an improvement.

File: app/views/flights/show.html.erb (hunk 2 above)
Comment (nmking22) [reply app/views/flights/show.html.erb:72]: Good catch! Fixed

**Pull request merged**